lib/krill.js

```diff
@@ -470,6 +524,28 @@ function krillPrimValidateFieldType(fieldtypes, pred, key)
 
 }
 
+function sanityCheck(pred) {
```

Comment:
I think it would be useful to document this a little.  Just reading through the file, it's not clear when this is called or what it's checking.